apisix/plugins/openid-connect.lua

```diff
@@ -471,7 +472,8 @@ local function required_scopes_present(required_scopes, http_scopes)
 end
 
 function _M.rewrite(plugin_conf, ctx)
-    local conf = core.table.clone(plugin_conf)
+    local conf_clone = core.table.clone(plugin_conf)
```

Comment:
performance tips:

we can use a lrucache, avoid repeated rendering of plugin_conf configuration